MakeLensPop.py

```diff
@@ -1,250 +1,18 @@
 import distances
-from scipy import interpolate
-import cPickle,numpy,math
-import indexTricks as iT
-import pylab as plt
-from PopulationFunctions import *
+import lenspop
```

Comment:
`MakeLensPop.py` is now a simple test script that calls some `lenspop` classes and computes a lens population. All the classes etc are in the `lenspop` folder.